packages/core/shared/src/nodes/io/SpellByName.ts

```diff
@@ -124,11 +127,14 @@ export class SpellByName extends MagickComponent<Promise<ModuleWorkerOutput>> {
 
     const spellId = firstSpell.id || firstSpell._id
 
+    const spellInputs = inputs.inputs[0] as Record<string, unknown>
+
     const { projectId } = _context
     if (agent) {
       const runComponentArgs = {
         inputs: {
-          'Input - Default': event,
```

Comment:
No, this is just how the SpellByName component is invoking its subspell.  This will trigger off the subspell event input, which is the main way right now to pass a trigger event down into the subspell to kick off.